src/main/java/pro/sky/recipes/services/impl/RecipeServiceImpl.java

```diff
@@ -52,9 +54,9 @@ public boolean deleteRecipe(int id) {
 
     @Override
     public RecipeDTO getAllRecipe() {
-        for (Recipe recipes : recipeMap.values()){
-            Recipe recipe = recipeMap.get(id);
+        List<RecipeDTO> recipeDTOList = new ArrayList<>();
+        for (RecipeDTO recipes : recipeDTOList) {
         }
-        return (RecipeDTO) recipeMap;
+        return (RecipeDTO) recipeDTOList;
```

Comment:
@Override
    public List<RecipeDTO>  getAllRecipe() {
        List<RecipeDTO> list = new ArrayList<>();
        for (Recipe recipes : recipeMap.values()){
           list.add(RecipeDTO.from(ingredients));
        }
        return list;
}